### PR TITLE
Added support for ThrowableToStringMapper from PHPUnit 10

### PR DESCRIPTION
**Pull request by mnocon**
My attempt at solving https://github.com/Behat/Behat/issues/1421

This is my first contribution to Behat, so if I've missed anything please let me know 🙇

### CODE REVIEW

**Comment by ciaranmcnulty** [comment]
Wow, I'd forgotten this existed - I wonder if we should add some composer restrictions to e.g. disallow PHPUnit 11?

File: src/Behat/Testwork/Exception/Stringer/PHPUnitExceptionStringer.php [hunk 1]
```diff
@@ -35,7 +35,11 @@ public function supportsException(Exception $exception)
      */
     public function stringException(Exception $exception, $verbosity)
     {
-        if (!class_exists('PHPUnit\\Framework\\TestFailure')) {
+        if (class_exists('PHPUnit\\Util\\ThrowableToStringMapper')) {
+            return trim(\PHPUnit\Util\ThrowableToStringMapper::map($exception));
+        }
+
+        if (class_exists('PHPUnit_Framework_TestFailure')) {
```
Comment (stof) [inline src/Behat/Testwork/Exception/Stringer/PHPUnitExceptionStringer.php:42]: this changed the behavior when both `PHPUnit_Framework_TestFailure` and `PHPUnit\Framework\TestFailure` exist.

**Comment by krzysztofrewak** [comment]
Hi, @stof! Could you rereview @mnocon changes? It would help me -and probably others as well - a lot. Thanks! :)

**Review by krzysztofrewak — APPROVED** [review]

**Review by stof — APPROVED** [review]

**Comment by arnidan** [comment]
Hello @everzet!

Could you review and merge the PR? 

Thanks

**Comment by lgnap** [comment]
Interested also in the possibility to merge that ;-) 

Thanks

**Comment by ciaranmcnulty** [comment]
Thanks @mnocon I'll see about making a release in the next few days

**Pull request merged**